src/Command/ColdStorageFlagCommand.php

```diff
@@ -78,78 +73,44 @@ protected function configure()
-
-                    // Set filesize, hash, and original filename. (Cold Storage attributes).
-                    $datasetSubmission->setDatasetFileColdStorageAttributes($size, $hash, $originalFilename);
-
-                    // Set options for a new replacement datafile of the supplied Cold-Storage stubfile.
-                    $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_NONE);
-                    $datasetSubmission->setDatasetFileName(null);
-                    $datasetSubmission->setDatasetFileSha256Hash(null);
-                    $datasetSubmission->setDatasetFileTransferType(DatasetSubmission::TRANSFER_TYPE_SFTP);
-                    $datasetSubmission->setDatasetFileUri($stubFileName);
-
-                    $this->entityManager->persist($datasetSubmission);
-                    $this->entityManager->flush();
-
-                    //Use rabbitmq to process dataset file and persist the file details. This will
-                    //Trigger filer and hasher (via filer) to complete the process.
-                    $datasetSubmissionFilerMessage = new DatasetSubmissionFiler($datasetSubmission->getId());
-                    $this->messageBus->dispatch($datasetSubmissionFilerMessage);
-                }
-            } else {
-                throw new \Exception("Could not open $stubFileName, expected to be at same location as $infoFileName.");
-            }
+        $io = new SymfonyStyle($input, $output);
+        $udi = $input->getOption('udi');
+        $originalFileSize = $input->getOption('originalfilesize');
+        $originalFileHash = $input->getOption('originalfilehash');
+        $originalFileName = $input->getOption('originalfilename');
+
+        $io->note("UDI: ($udi)");
+        $io->note("Original Size: ($originalFileSize)");
+        $io->note("Original Hash: ($originalFileHash)");
+        $io->note("Original File Name: ($originalFileName)");
+        $io->note("Attempting to flag $udi as Cold Stored.");
+
+        $dataset = $this->entityManager->getRepository(Dataset::class)->findOneBy(array('udi' => $udi));
+        if ($dataset instanceof Dataset) {
+            $io->note('Dataset Found.');
         } else {
-            throw new \Exception("Error: Could not open $infoFileName.");
+            throw new \Exception('Could not find a dataset with the udi provided.');
         }
 
+        $datasetSubmission = $dataset->getDatasetSubmission();
+        if (!($datasetSubmission instanceof DatasetSubmission)) {
+            throw new \Exception('Could not find Dataset Submission.');
+        } else {
+            $io->note('Submission Found.');
+
+            $datasetSubmission->setModifier($systemPerson);
+            $datasetSubmission->setDatasetFileColdStorageAttributes($originalFileSize, $originalFileHash, $originalFileName);
+
+            $this->entityManager->persist($datasetSubmission);
+            $this->entityManager->flush();
+        }
         return 0;
```

Comment:
I think we could with with a success message here: (It was a bit confusing, this was it really says it's done.
```suggestion

        $io->success('Done!');
        
        return 0;
```